CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 2.14.0 (2018-11-29)
+- Added `stache-hide-from-search` component to allow users to remove content from Stache Search results. [#557](https://github.com/blackbaud/stache2/pull/557)
+- Fixed alignment issue with sidebar and table of contents alignment. [#556](https://github.com/blackbaud/stache2/pull/556)
+- Updated version of `skyux-builder`.
```

Comment:
Elsewhere in the changelog, we just say "SKY UX Builder" with that capitalization and without the code snippet formatting. I think we should use that style here as well unless there is a reason to format differently in this instance. 

Also, should there be a link to a PR here?

Reply:
The ones that don't have a direct link to a PR are all included in the following entries. There's two instances in this update where one PR fixed several things.

Reply:
Changed the formatting to clarify this